packages/runner-ct/cypress/component/App.spec.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react'
+import { mount } from '@cypress/react'
+import Container from '../../src/app/container'
+import State from '../../src/lib/state'
+
+type ContainerProps = { config: Cypress.ResolvedConfigOptions, state: State};
+// This test suite is using specs from ../dummy_spec/* folder.
+// It runs the root of runner-ct with special config and specs to mimic e2e tests
+// Everything is bad: This file can contain only one test, cause cypress moves the focus to the internal test.
+// If we fix this – we can move to the real e2e testing painless
+describe('Root testing', () => {
+  let specs: Cypress.Spec[]
+
+  before(() => {
+    cy.task('getDummySpecs')
+    .then((dummySpecs) => {
+      specs = dummySpecs as any
+
+      cy.fixture('test1.html').then((html) => {
+        specs.forEach((spec) => {
+          cy.intercept(`/__cypress/iframes/${spec.relative}`, (res) => {
+            res.reply((req) => {
+              req.headers['content-type'] = 'text/html'
+              req.body = html
+            })
+          })
+        })
+      })
+    })
+  })
+
```

Comment:
Interesting way to test - we definitely get more coverage this way. I did something a little different - more light weight but also much less coverage: https://github.com/cypress-io/cypress/pull/14856/files#diff-91220e02fd6d9784254ef73471dd6a775476cf9447bd2c390aeb16a0ab7b62eb

I did not need to run a spec in my test, though.